### DISCUSSION TITLE
Authoring Platform : Missing Public Repositories and Jira Permission schema

### DISCUSSION THREAD

**Discussion by someshwarMirge** [IHTSDO/snowstorm#518] [Q&A]
Hi, 

We are deploying Authoring Platform in our facility. We have deployed available services and concept Authoring is working on it. We are not able to find repositories for following services and UI: 

**1. Release Notes Management UI 
2. Release Management Service 
3. Snomed Release Dashboard** 

Also, while going through Authoring UI, we came across multiple user roles for which Jira Permission schema is not provided. Following roles are missing the schema: 

**1. Project Manager 
2. Project Lead
3. Release Lead** 

We need help regarding: 
**1.  Missing Repositories 
2. schema for User roles** 

Please provide any other information required for the release process.
Thank you!

**Comment by kaicode** [comment]
Apologies for not finding this earlier!

Links to requested projects:
- [Release Notes API](https://github.com/IHTSDO/release-notes-service) and [UI](https://github.com/IHTSDO/release-notes-service-ui).
- [SNOMED Release Service](https://github.com/IHTSDO/snomed-release-service) and [Release Dashboard UI](https://github.com/IHTSDO/Release-Dashboard-UI).

## Release RBAC
This is a copy of our documentation on user roles related to the release service.

- There is a requirement for users to be added to ihtsdo-sca-author group to enable them to view their relevant Release Centres.

### Role levels
The users have access to the release dashboard via the RBAC functionality, which provides for different levels of access depending on the Role group assigned to the user:

- **USER ("RELEASE_USER")**
  - Provides read-only access to certain Release Centre(s) in the RAD, depending on the User's group access rights. 
  - So far no requirement to restrict between different Builds or Build types (Pre-Prod, Published, etc) - users who have access to a Release Centre should be able to view all Builds + Build Types
  - Users do NOT have access to create new Release Centres, Delete Builds, Publish any Releases, or see other Release Centres/Builds.
- **RELEASE LEAD ("RELEASE_LEAD")**:**
  - ALL RELEASE_USER PERMISSIONS +
  - Provide access to Products + Builds:
    - add
    - configure
    - validate
    - delete (BUT only builds that are not "PUBLISHED")
- **RELEASE MANAGER ("RELEASE_MANAGER"):**
  - ALL RELEASE_LEAD PERMISSIONS +
  - Publishing of any Build 
- **ADMIN ("RELEASE_ADMIN"):**
  - ALL RELEASE_MANAGER PERMISSIONS +
  - Creation of new Release Centres
  - Ability to add the default codesystem (to pull the permissions from Snowstorm)
- **NB FOR EACH ROLE LEVEL**, Users' permissions can be set either
  - globally, to provide access to ALL Release Centres (for developers, devOps, etc), or
  - per codesystem (to specific Release Centre(s)), to allow local Release Leads to access only their own Release(s) (for example, for Extension Managers, MS users, suppliers, etc).
